specifyweb/frontend/js_src/lib/components/Notifications/hooks.tsx

```diff
@@ -29,6 +29,9 @@ export function useNotificationsFetch({
   const [notifications, setNotifications] = React.useState<
     RA<GenericNotification> | undefined
   >(undefined);
+
+  const addSinceToNotificationRef = React.useRef(true);
```

Comment:
don't you need to set it to false by default?
i.e don't add "since" on first fetch so that it fetched all?